frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -34,6 +34,13 @@ import (
 	"github.com/hashicorp/go-multierror"
 )
 
+type BuildAPICredentials struct {
+	AccessToken string
+	// The credential for exchanging access tokens should be generated from a GCP project that
+	// enables the Build API. If it isn't, CredsUserProjectID is required for successful API usage.
```

Comment:
state what's is the expected value for the field `CredsUserProjectID` when you need to set it, i.e the `GCP project id the AccessToken was created with`.